src/commands/suggestions/utils.ts

```diff
@@ -1,16 +1,11 @@
 import { suggestionStatesReadable, getAvailableStatesString } from '../../components/suggestions';
 
-export const validateUpdate = (value: string): boolean | string => {
+export const validateIDs = (value: string): boolean | string => {
   // Split by white space
   const values = value.split(' ');
-  const state = values[0];
-  // validate if first word is one of the available states
-  if (!(state.toLowerCase() in suggestionStatesReadable)) {
-    return `you entered an invalid state. Please enter one of ${getAvailableStatesString()}.`;
-  }
 
   // validate each id after first word
-  for (let i = 1; i < values.length; i++) {
+  for (let i = 0; i < values.length; i++) {
     if (isNaN(Number(values[i]))) {
```

Comment:
you can use lodash `some` instead of a loop here
```
if (_.some(values, isNaN)) {
  return `you entered an invalid id: ${values[i]}. Please enter numbers only.`;
}
```

Reply:
If you really want, this function can be a one liner
```
export const validateIDs = (value: string): boolean | string =>
  _.some(values.split(' '), isNaN) ? `you entered an invalid id: ${values[i]}. Please enter numbers only.` : true
```
Just make sure to add a comment about what it's doing.